maraboupy/Marabou.py

```diff
@@ -74,6 +76,20 @@ def read_onnx(filename, inputNames=None, outputNames=None, reindexOutputVars=Tru
     """
     return MarabouNetworkONNX(filename, inputNames, outputNames, reindexOutputVars=reindexOutputVars)
 
+def read_onnx_with_threshould(filename, inputNames=None, outputNames=None, reindexOutputVars=True, threshold=None):
+    """Constructs a MarabouNetworkComposition object from an ONNX file
+
+    Args:
+        filename (str): Path to the ONNX file
+        inputNames (list of str, optional): List of node names corresponding to inputs
+        outputNames (list of str, optional): List of node names corresponding to outputs
+        threshold (int, optional): Threshold for the number of linear equations
```

Comment:
A more readable name for this would be `max_number_of_lin_equations` or something similar. The current name is really not very meaningful, and the comment doesn't clear anything up. What happens when you cross the threshold?

It would also be good to provide in the docs *why* you would want to limit the number of linear equations?

Reply:
done